web/satellite/src/utils/stripe.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+export default class Stripe {
```

Comment:
lets add some comments here

Reply:
this class is not a view and it shouldn't know anything about DOM. All DOM stuff like html elements, events, etc should be placed in view

Reply:
All DOM was moved here to prevent code duplication on each stripe elements integration